src/zope/app/apidoc/codemodule/browser/README.rst

```diff
@@ -44,7 +44,7 @@ Return info objects for all classes in this module.
   >>> pprint(details.getClasses())
   [{'doc': 'Represent the code browser documentation root',
     'name': 'CodeModule',
-    'url': 'http://127.0.0.1/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
+    'url': 'http://127.0.0.1/++etc++site/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
```

Comment:
Oh, also, when it always reparents, that's when we run into https://github.com/zopefoundation/zope.securitypolicy/issues/8 (apparently it creates circular parent chains doing that). That doesn't happen if the parent is only set if needed.